lib/tasks/test_vmdb.rake

```diff
@@ -4,7 +4,7 @@ if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
 namespace :test do
   namespace :vmdb do
     desc "Setup environment for vmdb specs"
-    task :setup => [:initialize, :verify_no_db_access_loading_rails_environment] do |rake_task|
+    task :setup => :initialize do |rake_task|
```

Comment:
I don't understand removing this - We should run this locally as well

Reply:
this is setting the database environment and we are having a hard time undoing these changes. I'd prefer to get rails running first and then worry about this automatically running.